code/game/objects/items/weapons/sword_of_truth.dm

```diff
@@ -4,6 +4,7 @@
 	icon = 'icons/obj/faction_item.dmi'
 	icon_state = "nt_sword_truth"
 	item_state = "nt_sword_truth"
+	force = WEAPON_FORCE_BRUTAL
```

Comment:
May conflict with https://github.com/discordia-space/CEV-Eris/pull/5956

Reply:
I don't see any conflicts showing up in the github, does that mean it's good?